test/fixtures/hdfs-fixture/build.gradle

```diff
@@ -44,4 +44,6 @@ dependencies {
   api "com.fasterxml.jackson.core:jackson-databind:${versions.jackson_databind}"
   api 'net.minidev:json-smart:2.4.8'
   api "org.mockito:mockito-core:${versions.mockito}"
+  api "com.google.protobuf:protobuf-java:3.21.2"
+  api "org.eclipse.jetty.websocket:websocket-client:9.4.48.v20220622"
```

Comment:
I believe this dependency could be excluded, there is no WS usage anywhere (afaik), right?